src/app/layout/Appbar/Appbar.js

```diff
@@ -1,12 +1,23 @@
 import PropTypes from "prop-types";
 import React from "react";
 import { Link } from "react-router-dom";
+import { useSelector } from "react-redux";
 
 import { ReactComponent as Logo } from "../../../img/logo.svg";
 import NavigationDrawer from "../NavigationDrawer/index";
-import { AppbarContainer, AppbarDefault, LogoContainer, MenuContainer } from "./Appbar.style";
+import {
+  AppbarContainer,
+  AppbarDefault,
+  LogoContainer,
+  MenuContainer,
+  AboutText,
+} from "./Appbar.style";
 
 const Appbar = ({ children }) => {
+  const isEmpty = useSelector((state) => state.firebase.auth.isEmpty);
+  function linkToAbout() {
+    window.location = "/about";
```

Comment:
I would rather use : {useHistory} from 'react-router-dom' and then do history.push("/about") instead of window.location.